packages/web3-eth/src/web3_eth.ts

```diff
@@ -1653,4 +1656,193 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 			notClearSyncing ? Web3Eth.shouldClearSubscription : undefined,
 		);
 	}
+
+	/**
+	 * Sends a UserOperation to the bundler. If accepted, the bundler will add it to the UserOperation mempool and return a userOpHash.
+	 *
+	 * @param UserOperation represents the structure of a transaction initiated by the user. It contains the sender, receiver, call data, maximum fee per unit of Gas, maximum priority fee, signature, nonce, and other specific elements.
+	 * @param entryPoint a singleton contract to execute bundles of UserOperations. Bundlers/Clients whitelist the supported entrypoint.
+	 * @param blockNumber ({@link BlockNumberOrTag} defaults to {@link Web3Eth.defaultBlock}) Specifies what block to use as the current state for the balance query.
+	 * @param returnFormat ({@link DataFormat} defaults to {@link DEFAULT_RETURN_FORMAT}) Specifies how the return data should be formatted.
+	 * @returns The current balance for the given address in `wei`.
```

Comment:
It returns user op hash or throws error instead of balance. 
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4337.md#return-value

Reply:
fixed, thx